docs/usage-advanced.md

```diff
@@ -12,9 +12,14 @@ To completely remove any frontmatter from exported notes, use `--frontmatter=nev
 
 ## Ignoring files
 
-By default, hidden files, patterns listed in `.export-ignore` as well as any files ignored by git (if your vault is part of a git repository) will be excluded from exports.
+The following files are not exported by default:
+* hidden files (can be adjusted with `--hidden`)
```

Comment:
Liberal markdown parsers will accept this, but it's better to have a blank line between the preceding paragraph and the start of the bullet list.

```suggestion
The following files are not exported by default:

* hidden files (can be adjusted with `--hidden`)
```